src/radical/pilot/configs/resource_xsede.json

```diff
@@ -48,6 +50,106 @@
+        "mpi_launch_method"           : "ORTE",
+        "pre_bootstrap_1"             : ["module purge",
+            "module load TACC",
+            "module load intel/15.0.2",
+            "module load python/2.7.9",
+            "module unload xalt",
+            # Load ICC license so we can build during bootstrap
+            "source ~train00/ssi_sourceme",
+            "export TACC_DELETE_FILES=TRUE",
+            "module use -a /home1/01740/marksant/openmpi/modulefiles",
+            "module load openmpi/rhc",
+            "export OMPI_MCA_plm=^slurm"
+        ],
+        "pre_bootstrap_2"             : ["export PATH=$PATH",
+            "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
+        ],
+        "default_remote_workdir"      : "$WORK",
+        "valid_roots"                 : ["/scratch", "$SCRATCH", "/work", "$WORK"],
+        "rp_version"                  : "local",
+        "virtenv"                     : "%(global_sandbox)s/ve_stampede",
+        "virtenv_mode"                : "create",
+        "python_dist"                 : "default"
+    },
+    "stampede_ortelib": {
+        "description"                 : "The XSEDE 'Stampede' cluster at TACC (https://www.tacc.utexas.edu/stampede/).",
+        "notes"                       : "Always set the ``project`` attribute in the ComputePilotDescription or the pilot will fail. To create a virtualenv for the first time, one needs to run towards the development queue.",
+        "schemas"                     : ["ssh", "gsissh", "go"],
+        "mandatory_args"              : ["project"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "slurm+ssh://stampede.tacc.utexas.edu/",
+            "filesystem_endpoint"     : "sftp://stampede.tacc.utexas.edu/"
+        },
+        "gsissh"                      :
+        {
+            "job_manager_endpoint"    : "slurm+gsissh://stampede.tacc.utexas.edu:2222/",
+            "filesystem_endpoint"     : "gsisftp://stampede.tacc.utexas.edu:2222/"
+        },
+        "go":
+        {
+            "job_manager_endpoint"    : "slurm+ssh://stampede.tacc.utexas.edu/",
+            "filesystem_endpoint"     : "go://xsede#stampede/"
+        },
+        "default_queue"               : "normal",
+        "lrms"                        : "SLURM",
+        "agent_type"                  : "multicore",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "ORTE",
+        "agent_launch_method"         : "ORTE_LIB",
+        "task_launch_method"          : "ORTE_LIB",
+        "mpi_launch_method"           : "ORTE_LIB",
+        "pre_bootstrap_1"             : ["module purge",
+            "module load TACC",
+            "module load intel/15.0.2",
+            "module load python/2.7.9",
+            "module unload xalt",
+            # Load ICC license so we can build during bootstrap
+            "source ~train00/ssi_sourceme",
+            "export TACC_DELETE_FILES=TRUE",
```

Comment:
it is.  Should we create a copy of that license file and use something like
```
export INTEL_LICENSE_FILE=/home1/our/shared/space//intel.current.20170224.lic
```
Is that license only required for ORTE, and why?